roles/matrix-postgres-backup/tasks/build_database_list.yml

```diff
@@ -0,0 +1,176 @@
+
+---
+
+# Detect wich databases have to backuped
```

Comment:
All of these tasks can probable be deleted.

This dynamic "list building" can also be redone in terms of `matrix_postgres_backup_databases` / `matrix_postgres_additional_databases` (discussed in another comment in this review)

Reply:
After the rework an use of group_vars we do not need this anymore. Removed.